src/FamilyHubs.ServiceDirectory.Web/Pages/ServiceFilter/_SummaryListRows.cshtml

```diff
@@ -2,14 +2,18 @@
 
 @model (string Key, IEnumerable<string> Values)
 
-<div class="govuk-summary-list__row">
-    <dt class="govuk-summary-list__key">
-        @Model.Key
-    </dt>
-    <dd class="govuk-summary-list__value">
-    @if (Model.Values.Any())
+@if (Model.Values.Any())
```

Comment:
Don't think this check is necessary, as if values is empty, it'll get caught by the next check below?